D/DomainSets/Compat.toml

```diff
@@ -1,4 +1,9 @@
 ["0.0"]
 IntervalSets = "0.3"
 StaticArrays = "0.8.3-0.10"
+
+["0.0.1"]
 julia = "0.7-1.3"
+
+["0.0.2"]
+julia = ["0.7", "1.3"]
```

Comment:
Does this release not work on Julia 1.0, 1.1, 1.2?

Reply:
That was a typo, fixed